kitsune/users/templatetags/jinja_helpers.py

```diff
@@ -37,12 +38,14 @@ def profile_avatar(user, size=48):
     except (Profile.DoesNotExist, AttributeError):
         avatar = settings.STATIC_URL + settings.DEFAULT_AVATAR
     else:
-        if profile.is_fxa_migrated:
+        if (profile.fxa_avatar and
+                requests.head(profile.fxa_avatar).status_code == 200):
```

Comment:
Why do you need this ?

Reply:
This adds potential i/o overhead to all helper calls.

Reply:
Also the default timeout is None which means that the request/response cycle relies on the availability of ``requests.head()``.

Reply:
The problem here is that while the user is logged in, the image can be removed from the management page of Firefox Accounts which has as a result a broken image everywhere where it's used.